packages/split-button/src/SplitButton/SplitButton.spec.tsx

```diff
@@ -174,7 +181,7 @@ describe('packages/split-button', () => {
     test('trigger opens the menu when clicked', () => {
       const { menuTrigger, getByTestId } = renderSplitButton({});
 
-      fireEvent.click(menuTrigger as HTMLElement);
+      fireEvent.click(menuTrigger);
```

Comment:
Nit: Leaving this as a nit because I see that we're using `fireEvent` already in previous tests here, but we should probably [prefer `userEvent` when possible](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event).
```suggestion
      userEvent.click(menuTrigger);
```

Reply:
Good candidate for a follow-up PR as this is simply removing a type assert and not proposing a name which wasn't already there.